catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -116,53 +116,78 @@ public void refreshRegistryEntries() throws FederationAdminException {
                 remoteResults.getRemoteRegistryMetacards();
         Map<String, Metacard> registryMetacardsMap = getRegistryMetacardsMap();
 
+        if (!remoteResults.getFailureList()
+                .isEmpty()) {
+            LOGGER.debug("Failed to query the following registry stores: {}",
+                    String.join(",", remoteResults.getFailureList()));
+        }
+
         List<Metacard> remoteMetacardsToUpdate = new ArrayList<>();
         List<Metacard> remoteMetacardsToCreate = new ArrayList<>();
         List<Metacard> remoteMetacardsToDelete = new ArrayList<>();
 
+        // Maps registry-id to a list of metacards that came from that registry
         Map<String, List<Metacard>> remoteRegistryToMetacardMap = getMetacardRegistryIdMap(
                 remoteRegistryMetacardsMap.values());
         Map<String, List<Metacard>> localRegistryToMetacardMap = getMetacardRegistryIdMap(
                 registryMetacardsMap.values());
 
+        // Loop through all the registry-ids of the stores queried and find any local internal
+        // representation metacards that need to be deleted
         for (String regId : remoteResults.getRegistryStoresQueried()) {
```

Comment:
Add TRACE logs to watch each registry be checked. Add TRACE statements after adding to each collection. Allows the user to see in real time to help identify exactly where errors might be occurring rather than waiting to the end and see that the results aren't correct. Can check if TRACE is enabled first.